### ISSUE TITLE
Differences between 9.0.x and 9.1.x, 2 peers as initators ?

### ISSUE THREAD

**Issue by G-Ray** [feross/simple-peer#429]
Please see the issue we encounter here: https://github.com/mafintosh/webrtc-swarm/issues/37

It fails when we got 2 peers trying to connect to each other as initatiator. However, there were no such issue with `simple-peer <= 9.0.x`. Can you help us find what changes between simple-peer `v9.0.x` and `v9.1.0` could explain that issue ?

**Comment by t-mullen** [comment]
Do you have a commit hash for a version that works and a later one that doesn't work, so we can bisect?

**Comment by t-mullen** [comment]
Does db50ec6 or 046039d work for you?

https://github.com/feross/simple-peer/releases

**Comment by G-Ray** [comment]
Thanks you @t-mullen . v9.0.0 works, meanwhile v9.1.0 does not.

**Comment by t-mullen** [comment]
Can you please try the latest published version 9.2.0?

**Comment by sujeebant** [comment]
We also had the same issue with the 9.2.0.
Is the connection between 2 peers trying to connect as initiators supposed to work ?

I think the error was caused by having two instances of simple-peer on each actual peer.
One instance of simple-peer trying to connect as non-initator and at the same time another instance of simple-peer trying to connect as initiator.

We fixed the issue in webrtc-swarm by having only one of the peers try to connect as initiator [on this branch of webrtc-swarm](https://github.com/QwantResearch/webrtc-swarm/tree/fix).
